commcare-cloud-bootstrap/bootstrap.sh

```diff
@@ -14,4 +14,4 @@ done
 commcare-cloud $ENV bootstrap-users --quiet --branch=$BRANCH
 commcare-cloud $ENV deploy-stack --skip-check --quiet -e 'CCHQ_IS_FRESH_INSTALL=1' --branch=$BRANCH
 
-commcare-cloud $ENV fab deploy:confirm=no,skip_record=yes --show=debug --set ignore_kafka_checkpoint_warning=true --branch=$BRANCH
+commcare-cloud $ENV fab deploy:confirm=no,skip_record=yes --show=debug --set ignore_kafka_checkpoint_warning=true
```

Comment:
Maybe this is from a mistake during the merge? I think you do want `--branch` here, right?

Reply:
Looks like file was in conflict, fixed here: 
https://github.com/dimagi/commcare-cloud/pull/1994/commits/a2add1d0e7ebf16ecc5ad5002da3b1768206cfa4